src/main/java/org/jvnet/hudson/plugins/repositoryconnector/aether/Aether.java

```diff
@@ -80,7 +76,7 @@ public Aether(Collection<Repository> remoteRepositories, File localRepository) {
 
     public Aether(Collection<Repository> remoteRepositories, File localRepository, PrintStream logger, boolean extendedLogging,
             String snapshotUpdatePolicy, String snapshotChecksumPolicy, String releaseUpdatePolicy, String releaseChecksumPolicy) {
-        this.logger = logger;
+        this.logger = logger==null?System.out:logger;
```

Comment:
i don't remember ...  but i wouldn't do this kind of thing out of the blue, i guess.